database/migrations/2024_08_16_104600_mobitech_expand_columns.php

```diff
@@ -0,0 +1,62 @@
+     */
+    public function up(): void
+    {
+        Schema::table('mobitech_transactions', function (Blueprint $table) {
+            $table->bigInteger('line_id')->comment('Skyttel (sales place) line ID')->change();
+            $table->bigInteger('actor_id')->comment('Skyttel (sales place) actor/operator ID')->change();
+            $table->bigInteger('tour_id')->comment('Skyttel (transaction trip) tour ID')->change();
+            $table->bigInteger('stop_place_id_entry')->comment('Skyttel stop place ID (start)')->change();
+            $table->bigInteger('stop_place_id_exit')->comment('Skyttel stop place ID (exit)')->change();
+            $table->char('tariff_class')->change();
+        });
+
+        Schema::table('mobitech_statistics', function (Blueprint $table) {
+            $table->bigInteger('actor_id')->comment('Skyttel (sales place) actor/operator ID')->change();
+            $table->bigInteger('line_id')->comment('Skyttel (sales place) line ID')->change();
+            $table->bigInteger('tour_id')->comment('Skyttel (transaction trip) tour ID')->change();
+            $table->bigInteger('stop_place_id_entry')->comment('Skyttel stop place ID (start)')->change();
+            $table->bigInteger('stop_place_id_exit')->comment('Skyttel stop place ID (exit)')->change();
+        });
+
+        Schema::table('mobitech_actors', function (Blueprint $table) {
+            $table->bigInteger('id')->comment('Unique actor ID')->change();
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     */
+    public function down(): void
+    {
+        Schema::table('mobitech_transactions', function (Blueprint $table) {
+            $table->integer('line_id')->comment('Skyttel (sales place) line ID')->change();
+            $table->integer('actor_id')->comment('Skyttel (sales place) actor/operator ID')->change();
+            $table->integer('tour_id')->comment('Skyttel (transaction trip) tour ID')->change();
+            $table->integer('stop_place_id_entry')->comment('Skyttel stop place ID (start)')->change();
+            $table->integer('stop_place_id_exit')->comment('Skyttel stop place ID (exit)')->change();
+            $table->char('tariff_class', 16)->change();
+        });
+
+        Schema::table('mobitech_statistics', function (Blueprint $table) {
+            $table->integer('actor_id')->comment('Skyttel (sales place) actor/operator ID')->change();
+            $table->integer('line_id')->comment('Skyttel (sales place) line ID')->change();
+            $table->integer('tour_id')->comment('Skyttel (transaction trip) tour ID')->change();
+            $table->integer('stop_place_id_entry')->comment('Skyttel stop place ID (start)')->change();
+            $table->integer('stop_place_id_exit')->comment('Skyttel stop place ID (exit)')->change();
+        });
+
+        Schema::table('mobitech_actors', function (Blueprint $table) {
+            $table->integer('id')->comment('Unique actor ID')->change();
+        });
```

Comment:
This is unneeded. The migration (`up()`) will allways succeed anyway.

Also, the man himselv — Taylor Otwell — has stated that [rollback or down() is not needed](https://laraveldaily.com/post/still-need-migrations-taylor-says-no). The world is always going forward.

The only reason I can see to have a rollback is during development and staging.